src/content/api/compiler-hooks.md

```diff
@@ -9,11 +9,12 @@ contributors:
   - misterdev
   - EugeneHlushko
   - superburrito
+  - wizardofhogwarts
 ---
 
 The `Compiler` module is the main engine that creates a compilation instance
 with all the options passed through the [CLI](/api/cli) or [Node API](/api/node). It extends the
-`Tapable` class in order to register and call plugins. Most user facing plugins
+`Tapable` class in order to register and call plugins. Most user-facing plugins
```

Comment:
Do you have a link or something that explains these dashing examples?

Reply:
https://www.grammarbook.com/punctuation/hyphens.asp